src/app/workspace/cmdinput/aichat.tsx

```diff
@@ -91,17 +92,18 @@ class AIChat extends React.Component<{}, {}> {
                     e.target.setRangeText("\n", e.target.selectionStart, e.target.selectionEnd, "end");
                 }
             }
-            if (e.code == "Escape") {
+            if (checkKeyPressed(waveEvent, "Escape")) {
                 e.preventDefault();
                 e.stopPropagation();
                 inputModel.closeAIAssistantChat();
             }
-            if (e.code == "KeyL" && e.getModifierState("Control")) {
+
```

Comment:
maybe remove space here
Idk what is your thought on spacing?
Apple was super strict on spacing inside of functions for some reason, so I'm hardwired to get rid of them